README.md

```diff
@@ -158,6 +160,7 @@ There are two mechanisms for displaying a launch screen on iOS:
 Apple is moving away from legacy launch images. There is no official support for providing a native-resolution launch image for the iPad Pro 12.9 or for providing launch images that work with split-screen multitasking or slide-over. If your app doesn't need to support these contexts, then you can continue to use legacy launch images for as long as you like. 
 
 The preferred method of providing launch images is to use a launch storyboard. For native app developers, the ideal launch storyboard is an unpopulated version of the app's user interface at launch. For non-native app developers who don't wish to learn Interface Builder, however, this plugin simulates the legacy launch image method as much as is feasible.
+> Note: Since ios 11, for iphone x devices and greater (with notch screen) you should switch to the new storyboard splash screens, taking into account to add viewport-fit=cover to the viewport meta tag in your index.html file to display the app correctly, and make some modification to your app style (css file) by using: padding: env(safe-area-inset-top) to avoid the "unsafe" areas behind notches in the screen.
```

Comment:
Thanks for your PR, just a few things to keep a consistent formatting.

"Note:" should not use the block-quote formatting. Note: should be bolded e.g... "**Note:** Since iOS 11..."

"ios" should be "iOS"

"iphone x" should be "iPhone X"

"index.html" should be enclosed in back-ticks, `index.html`

"padding: env(safe-area-inset-top)" should be enclosed in back-ticks, `padding: env(safe-area-inset-top)`

Reply:
Thanks for info